pipert2/core/base/pipe.py

```diff
@@ -46,31 +43,22 @@ def __init__(self, event_board: EventBoard = EventBoard(), network: Network = Qu
         self.routines_dict = {}
         self.event_board = EventBoard()
         self.default_data_transmitter = data_transmitter
-        self.run_rpc_cli = run_rpc_cli
         self.flows = {}
         self.wires: Dict[tuple, Wire] = {}
 
-        if self.run_rpc_cli:
-            self.rpc_server = RPCPipeWrapper(notify_callback=self.event_board.get_event_notifier())
-        else:
-            self.rpc_server = None
-
         if auto_pacing_mechanism:
             self.routine_synchroniser = RoutinesSynchroniser(event_board=self.event_board,
                                                              notify_callback=self.event_board.get_event_notifier())
         else:
             self.routine_synchroniser = None
 
-    def run_rpc_server(self, endpoint: str):
-        """Binds it to a given endpoint and runs the rpc server.
+    def get_event_notify(self) -> callable:
+        """Get callable for the event notify function.
 
-            Arguments:
-                endpoint: server's endpoint
+        Returns:
+            Callable for the event notify function.
         """
-        if self.rpc_server is None:
-            raise TypeError
-
-        self.rpc_server.run_rpc_server(endpoint=endpoint)
+        return self.event_board.get_event_notifier()
```

Comment:
Why not using the pipe notify event function?

Reply:
When I asked you/Elon you said that You did the get_event_notifier because of the multiprocessing and the input_queue instance, so get_event_notifier is process safe.